packages/lib-classifier/src/plugins/drawingTools/components/DrawingToolRoot/DrawingToolRoot.js

```diff
@@ -55,6 +56,10 @@ const DrawingToolRoot = forwardRef(function DrawingToolRoot({
     onDeselect(mark)
   }
 
+  let transform = ''
+  transform = (mark.x && mark.y) ? `${transform} translate(${mark.x}, ${mark.y})` : transform
+  transform = mark.angle ? `${transform} rotate(${mark.angle})` : transform
```

Comment:
There should be a spec for this

Reply:
Good point.

Reply:
Turns out the drawing tool root doesn't have any real tests at all, yet.

Reply:
#1410 adds specs for `DrawingToolRoot`.

Reply:
I'll cherry pick those onto this branch and add tests for `transform`